cmd/server.go

```diff
@@ -43,7 +43,7 @@ func NewServeCmd(stdin io.Reader, stdout, stderr io.Writer) *cobra.Command {
 		Long: `pilosa server runs Pilosa.
 
 It will load existing data from the configured
-directory, and start listening client connections
+directory and start listening for client connections
 on the configured port.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			logOutput, err := server.GetLogWriter(Server.Config.LogPath, stderr)
```

Comment:
so, I think that if this actually opens a file, that file won't get closed when Pilosa exits. GetLogWriter is also called inside Command.Run which is kinda weird - that file might get opened twice? I don't think it gets closed either time right now. 

I think GetLogWriter should save the logger on the server.Command, and be responsible for closing any file it may have opened when Command.Close is called.